src/_locales/en/messages.json

```
@@ -24,6 +24,21 @@
   "siteNotMonetized": {
     "message": "This website is not monetized."
   },
+  "error_keyRevoked_title": {
+    "message": "Unauthorized access to wallet"
+  },
+  "error_keyRevoked_text": {
+    "message": "It appears that the key has been revoked from your wallet. Authenticate wallet by adding the key again, or disconnect."
+  },
+  "action_keyRevoked_disconnect": {
+    "message": "Yes, let me disconnect my wallet"
+  },
+  "action_keyRevoked_reconnect": {
+    "message": "It was a mistake, I’d like to reconnect my wallet"
+  },
```

I feel like we are being to wordy for these two. Thoughts on:
```suggestion
  "action_keyRevoked_disconnect": {
    "message": "Disconnect my wallet"
  },
  "action_keyRevoked_reconnect": {
    "message": "Reconnect my wallet"
  },
```
CC @RabebOthmani

---

For reference: [this part in Figma](https://www.figma.com/design/0S8Oj8ZB5MvkGtIrVRswDH/Web-monetization?node-id=474-413&m=dev)